packages/odyssey-storybook/src/components/odyssey-mui/Dialog/Dialog.stories.tsx

```diff
@@ -34,29 +26,38 @@ export default {
     },
   },
   argTypes: {
-    content: {
-      control: "text",
-      defaultValue: "The mission to Sagitarius A has been set for January 7.",
+    actions: {
+      control: null,
+      defaultValue: (
+        <>
+          <Button variant="floating">Cancel</Button>
+          <Button variant="primary">Primary action</Button>
+        </>
+      ),
```

Comment:
Can we define this on a story template?

I just remembered, these default values are supposed to be for documentation, not for setting defaults.

They're supposed to be auto-generated and represent the default value in your component, not one we make up here.

Oh man... We're gonna have to go back and review our old stories 😨. Let's get a ticket written up for this work. It's part of our cleanup tasks.